Packs/ProofpointCasb/README.md

```diff
@@ -0,0 +1,40 @@
+# Proofpoint CASB
```

Comment:
Add to the README a section that mentions the timestamp field you extract and the format you support.